app/merginapi.h

```diff
@@ -21,6 +21,8 @@ Q_ENUMS( ProjectStatus )
 struct MerginProject
 {
   QString name; //projectNamespace + projectName
```

Comment:
the comment does not seem to be right - it is just project name without namespace...?